src/mockServiceWorker.js

```diff
@@ -243,6 +243,11 @@ If you wish to mock an error response, please refer to this guide: https://mswjs
 self.addEventListener('fetch', function (event) {
   const { request } = event
 
+  // Bypass server-sent events.
+  if (request.headers.get('accept') === 'text/event-stream') {
```

Comment:
Should we account for the cases when the `accept` header includes `text/event-stream` in a list of accepted content types? 

```
Accept: text/plain; text/event-stream;
```

That should still be possible per specification.

Perhaps something like this?

```js
const accept = request.headers.get('accept') || ''

if (accept.includes('text/event-stream')) {}
```

What do you think about this, @takefumi-yoshii?